### ISSUE TITLE
Performance regression with Metro: Last Light Redux on 3.5

### ISSUE THREAD

**Issue by Glorydayreject** [ValveSoftware/SteamOS#1266]
Steam Deck 1TB Limited Edition 

* Steam client version: 1701289036
* SteamOS version: 3.5.7
* Opted into Steam client beta?: [No] 
* Opted into SteamOS beta?: [No] 
* Have you checked for updates in Settings > System?: [Yes]

After launching the game Metro: Last Light Remastered I am experiencing performance issues that were not present in SteamOS version 3.4.11.
When moving around I am experiencing stuttering as well as the GPU switching between 200 and 1040 Mhz. 
When idle, the GPU speed increases. 

**Settings:**
TDP Limit - off
Manual Clock GPU - off
Allow Tearing - off
FSR - off
Refresh Rate - 45 FPS / 90Hz

The only fix that seems to curb the stuttering is manually setting the GPU clock speed to 1600Mhz and leaving TDP off. 
Previously, I could leave TDP at 11 and maintain 45 FPS, while leaving GPU clocking on auto.

I am now also experiencing this while on the homepage, nothing running in the background. Mostly running at 1040 Mhz on homepage. 

#### Steps for reproducing this issue:

1. Installed Metro on SD card
2. Capped display to 45 FPS/90 HZ
3. Launched game and continued where I left off 
4. Rotate while standing in place

Video of Metro GPU issue - https://drive.google.com/file/d/1HKXJ1YamlfuvIZa8MPa3lnKZKRL4JdJt/view?usp=drivesdk

Screenshot of homepage - https://drive.google.com/file/d/1HL24Pa9JNasY0i1Ec2ANDMRFCLnVcWxX/view?usp=drivesdk

**Comment by ruipt-94** [comment]
@lostgoat

**Comment by Glorydayreject** [comment]
@lostgoat Yes the redux. I thought redux included 2033 and Last Light so I just said remastered

**Comment by Glorydayreject** [comment]
@lostgoat 
Here's another video - https://drive.google.com/file/d/1IH80eX_m51GxWfF9RPQ7L6AzG_SAnv4s/view?usp=drivesdk

The framerates are "stable" but you can see the stuttering when I past a certain part. You can also see the graph change when the stuttering occurs. 

Once I switch the GPU to 1600 Mhz, the stuttering is gone and the graph is a perfect straight line

**Comment by Glorydayreject** [comment]
> @lostgoat Here's another video - https://drive.google.com/file/d/1IH80eX_m51GxWfF9RPQ7L6AzG_SAnv4s/view?usp=drivesdk
> 
> The framerates are "stable" but you can see the stuttering when I past a certain part. You can also see the graph change when the stuttering occurs.
> 
> Once I switch the GPU to 1600 Mhz, the stuttering is gone and the graph is a perfect straight line

@lostgoat 
Has there been any progress on this?

**Comment by jvalecillos** [comment]
Does this still happen on _SteamOS_ `3.5.17`?
